refine readme Visual C++ Build Environment text for clarity (#2965)
nodejs/node-gyp: README.md

[Microsoft Store](https://apps.microsoft.com/store/search?publisher=Python+Software+Foundation).

Install tools and configuration manually:

>    * Install Visual C++ Build Environment: If using a version older than Visual studio 2019, install [Visual Studio Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools) with the "Visual C++ build tools" option, For Visual Studio 2019 or later, use the "Desktop development with C++" workload. You can also use [Visual Studio Community](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community) with the "Desktop development with C++" workload

Reviewer: Let’s reverse this because increasingly Developers should be coming to this with tools released in the last 5 years…
```suggestion
   * Install Visual C++ Build Environment: For Visual Studio 2019 or later, use the `Desktop development with C++` workload from [Visual Studio Community](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community).  For a version older than Visual Studio 2019, install [Visual Studio Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools) with the `Visual C++ build tools` option.
```

Author: done that @cclauss
